ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh

```diff
@@ -67,7 +70,22 @@ then
   . network.sh
   . utils.sh
 
-  ssh -o 'ConnectTimeout=5' -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' -o 'ServerAliveInterval=90' core@\$BOOTSTRAP_PROVISIONING_IP TAR_FILE=/tmp/log-bundle-bootstrap.tar.gz sudo -E /usr/local/bin/installer-gather.sh --id bootstrap &&
+  # Pass master IPs to installer-gather script to collect info from nodes which didn't join the cluster
+  NODE_IPS=()
```

Comment:
I'm trying to think of a cleaner way to do this, not sure if it will work
Can we leverage BMH resources to get all those IPs? (or we're concerned about this being less low-level than interacting with libvirt)

Reply:
Yeah, there's a chance the IP will not be saved there. Lets use libvirt just to be sure

Reply:
Ack. So in this case that's indeed the best way to do it